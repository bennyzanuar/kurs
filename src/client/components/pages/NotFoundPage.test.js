import React from 'react'
import { shallow } from 'enzyme'
import NotFoundPage from './NotFoundPage'

describe('<NotFoundPage />', () => {
    it('should render not found page', () => {
        const component = shallow(<NotFoundPage />)
        expect(component).toMatchSnapshot()
    })
})