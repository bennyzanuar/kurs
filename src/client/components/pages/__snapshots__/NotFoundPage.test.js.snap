// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should render not found page 1`] = `
<h1>
  Ooops, route not found.
</h1>
`;
